### ISSUE TITLE
Missing TS module declaration 

### ISSUE THREAD

**Issue by vinceshere** [vue-youtube/vue-youtube#13]
![image](https://github.com/vue-youtube/vue-youtube/assets/8030457/7ad5af00-8dd5-451f-b54d-105e9f347cb8)

Package versions:
```
"vue": "3.3.4"
"typescript": "5.2.2"
"vite": "4.4.9"
```

To solve it, added a `shims-vue.d.ts` on the `src` folder of my project with the following content:

```
declare module '@vue-youtube/core' {
  import { createManager, usePlayer } from '@vue-youtube/core/dist/core/index'
  export { createManager, usePlayer }
}

declare module '@vue-youtube/component' {
  import YoutubeIframe from '@vue-youtube/component/dist/index'
  export { YoutubeIframe }
}
```

**Comment by vinceshere** [comment]
To solve it, adde a `shims-vue.d.ts` on the `src` folder of my project with the following content:

```
declare module '@vue-youtube/core' {
  import { createManager, usePlayer } from '@vue-youtube/core/dist/core/index'
  export { createManager, usePlayer }
}

declare module '@vue-youtube/component' {
  import YoutubeIframe from '@vue-youtube/component/dist/index'
  export { YoutubeIframe }
}
```

**Comment by Techassi** [comment]
This is related to #8. I'm still investigating the underlying issue. This should hopefully be fixed soon!

**Comment by Techassi** [comment]
Hey, this should now be fixed in version `0.0.6`. Let me know if everything works as expected!

**Comment by Techassi** [comment]
Let's close this for now. Feel free to re-open if you need any more help on this topic.
